modules/sdk-coin-ada/test/unit/utils.ts

```diff
@@ -11,11 +11,15 @@ describe('utils', () => {
     should.equal(Utils.default.isValidAddress(address.address3), false);
     should.equal(Utils.default.isValidAddress(address.address4), true);
     should.equal(Utils.default.isValidAddress('dfjk35y'), false);
+    let undefineVariable;
+    should.equal(Utils.default.isValidAddress(undefineVariable), false);
   });
 
   it('should validate block hash correctly', () => {
     should.equal(Utils.default.isValidBlockId(blockHash.hash1), true);
     should.equal(Utils.default.isValidBlockId(blockHash.hash2), true);
+    let undefineVariable;
+    should.equal(Utils.default.isValidBlockId(undefineVariable), false);
```

Comment:
NIT `undefineVariable` was coming in as any, suppressing the complaint, this should make a bit more clearer, and also add a small comment that it was a causing an error or something previously
```suggestion
    should.equal(Utils.default.isValidBlockId(undefined as unknown as string), false);
```